docker/build_container.sh

```diff
@@ -23,10 +23,10 @@ DOCKER_EXTRA_ARGS=${DOCKER_EXTRA_ARGS:-""}
 
 # Build args
 FROM_IMAGE=${FROM_IMAGE:-"gpuci/miniforge-cuda"}
-CUDA_VER=${CUDA_VER:-11.4}
+CUDA_VER=${CUDA_VER:-11.5}
 LINUX_DISTRO=${LINUX_DISTRO:-ubuntu}
```

Comment:
Again, does this need to be updated to centos?

Reply:
Again, I don't think this is necessary.